README.rst

```diff
@@ -1,35 +1,44 @@
-i18n Tools |build-status| |coverage-status|
-===========================================
+edX i18n Tools |build-status| |coverage-status|
+###############################################
 
 Installing
-----------
+==========
 
 ``python setup.py install``
 
 Running
--------
+=======
 
 Running commands from the edx-platform directory will default to loading the
 configuration at ``./conf/locale/config.yaml``. You can specify a different
 configuration file with the ``--config`` argument.
 
- * ``i18n_tool dummy``
- * ``i18n_tool extract``
- * ``i18n_tool generate``
- * ``i18n_tool segment``
- * ``i18n_tool transifex``
- * ``i18n_tool validate``
+* ``i18n_tool dummy``
+* ``i18n_tool extract``
+* ``i18n_tool generate``
+* ``i18n_tool segment``
+* ``i18n_tool transifex``
+* ``i18n_tool validate``
 
 
 Configuration
--------------
+=============
 
 Details of the config.yaml file are in `edx-platform/conf/locale/config.yaml
 <https://github.com/edx/edx-platform/blob/master/conf/locale/config.yaml>`_
 
 
+Changes
```

Comment:
"Changes" isn't empty, it has the "Unreleased" heading within it.  I used "Unreleased" because I don't know the version number and date that they will appear in yet.  It's the changes on master that haven't been tagged with a release number yet.